Kickstarter-iOS/Views/Cells/MostPopularCell.swift

```diff
@@ -23,6 +23,6 @@ internal final class MostPopularCell: UITableViewCell, ValueCell {
     _ = self.mostPopularLabel
       |> UILabel.lens.font .~ .ksr_title1(size: 22)
       |> UILabel.lens.textColor .~ .ksr_text_navy_700
-      |> UILabel.lens.text %~ { _ in Strings.search_most_popular() }
+      |> UILabel.lens.text %~ { _ in "Popular Projects"}
```

Comment:
let's wrap this in `localizedString` while we wait for the strings. that way if we `make strings` later and forget to update this we'll still get the translations.